docs/rules/no-tracked-properties-from-args.md

```diff
@@ -0,0 +1,34 @@
+# ember/no-tracked-properties-from-args
+
+<!-- end auto-generated rule header -->
+
+Disallow creation of @tracked properties from args.
+
+## Rule Details
+
+This rule disallows the creation of @tracked properties with values from `this.args`. The @tracked property will not be updated when the args change, which is almost never what you want. Instead, use a getter to derive the desired state.
+If you need to modify a specific arg, consider having the parent provide a way for the child component to update it. This avoids having two sources of truth that you will need to keep in sync.
+
+## Examples
+
+Examples of **incorrect** code for this rule:
+
+```js
+class Example {
+    @tracked someValue = this.args.someValue;
+}
+```
+
+Examples of **correct** code for this rule:
+```js
```

Comment:
blank line needed to pass markdownlint:

```suggestion

```js
```

Reply:
Thanks @bmish! Can I run all these linting checks locally? Apologies for the endless notifications you're likely getting from me fixing all of these 🙈

Reply:
Yes, `yarn lint` should work locally.